src/promptflow/promptflow/_sdk/operations/_run_submitter.py

```diff
@@ -266,7 +267,7 @@ def _run_bulk(self, run: Run, stream=False, **kwargs):
 
         # running specified variant
         with variant_overwrite_context(run.flow, tuning_node, variant, connections=run.connections) as flow:
-            local_storage = LocalStorageOperations(run, stream=stream)
+            local_storage = LocalStorageOperations(run, stream=stream, run_mode=RunMode.Batch)
```

Comment:
Even though it is duplicate, add a similar parameter in test_submitter ?

Reply:
Only batch runs require specifying the run mode.